src/test_verifier/main.cpp

```diff
@@ -122,6 +122,13 @@ int main(int argc, char** argv) {
                 }
             };
 
+            auto CheckAddress = [&](const char* name, u16 address, u16 expected, u16 actual) {
+                if (expected != actual) {
+                    std::printf("Mismatch: %s%s: %04X != %04X\n", name, (ToHex<u16>(address)).c_str(), expected, actual);
```

Comment:
Actually since we are using printf here we can just use `%04X` and avoid `ToHex`

Reply:
Addressed.

Also, I observed `ctest` didn't output anything as expected when tests are failed, and I will open a new PR to resolve that issue.